libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -975,6 +996,43 @@ void AP_GPS_UBLOX::log_rxm_rawx(const struct ubx_rxm_rawx &raw)
 }
 #endif // UBLOX_RXM_RAW_LOGGING
 
+#if UBLOX_RXM_RTCM_LOGGING
+void AP_GPS_UBLOX::log_rxm_rtcm(const struct ubx_rxm_rtcm &rtcm)
+{
+#if HAL_LOGGING_ENABLED
+    if (!should_log()) {
+        return;
+    }
+
+    uint64_t now = AP_HAL::micros64();
+
+    struct log_GPS_RTCM header = {
+        LOG_PACKET_HEADER_INIT(LOG_GPS_RTCM_MSG),
+        time_us    : now,
+        version : rtcm.version,
+        flags : rtcm.flags,
+        refStation : rtcm.refStation,
+        msgType : rtcm.msgType
+    };
+    AP::logger().WriteBlock(&header, sizeof(header));
+#endif
+}
+
+void AP_GPS_UBLOX::log_status(const struct ubx_nav_status &status)
+{
+#if HAL_LOGGING_ENABLED
```

Comment:
Remove this line too - should be aroun entire functions and the calls to them.

Reply:
Nope, wrong